src/core/iomgr/unix_sockets_posix_noop.c

```diff
@@ -35,7 +35,10 @@
 
 #ifndef GPR_HAVE_UNIX_SOCKET
 
-void grpc_create_socketpair_if_unix(int sv[2]) {}
+void grpc_create_socketpair_if_unix(int sv[2]) {
+  // TODO(ahedberg): Please fill in an implementation for non-Unix-sockets case
```

Comment:
Please provide more direction when adding TODOs assigned to volunteer contributors (e.g. me) than "fill in an implementation". This case was previously unhandled IIUC.